### ISSUE TITLE
Missing checkpoint loading in training code?

### ISSUE THREAD

**Issue by biubiubiiu** [LI-Hao-SJTU/DerainRLNet#4]
`_main_` shows that the model is trained in 3 stages, with the learning rate and loss function varying each time. There seems to be no checkpoint loading process in the training code, meaning we just train the model from scratch. Is this expected?

Code snippet in `derain`:

```python
netG1=net.Dense_rainall()
# ...
netG1=torch.nn.DataParallel( netG1, [0,1])
#netG1.load_state_dict(torch.load('./sample1/netG1_epoch_95.pth'))  # possible checkpoint loading here?

# start training...
```

**Comment by LI-Hao-SJTU** [comment]
Training from scratch is indeed expected.
